helix-term/src/application.rs

```diff
@@ -43,6 +43,8 @@ type Signals = futures_util::stream::Empty<()>;
 
 const LSP_DEADLINE: Duration = Duration::from_millis(16);
 
+pub static mut ENABLE_SIGTSTP: bool = true;
```

Comment:
you can use a Lazy once_cell to make mutating this safe

Reply:
Fixed. Thanks! I learned something 😄